mods/tts/src/plugin.ts

```diff
@@ -0,0 +1,39 @@
+import { SynthResult, TTSConfig, TTSPlugin } from "./types";
```

Comment:
Could you add the copyright notice to this file? Feel free to add a note including your contribution.